Gemfile

```diff
@@ -4,7 +4,7 @@ gem 'react_on_rails', path: '../react_on_rails'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.4'
+gem 'rails', '4.2.5'
```

Comment:
@justin808 that's the weird, thing, this is a diff between redux and redux-server-rendering, so it's nothing to do with the differences from the fresh-rails-install, right?